cmd/snap/main.go

```diff
@@ -201,8 +201,11 @@ snaps on the system. Start with 'snap list' to see installed snaps.`)
 		}
 
 		opts := cmd.Options()
+		for _, grp := range cmd.Groups() {
+			opts = append(opts, grp.Options()...)
+		}
 		if c.optDescs != nil && len(opts) != len(c.optDescs) {
-			logger.Panicf("wrong number of option descriptions for %s: expected %d, got %d", c.name, len(opts), len(c.optDescs))
+			logger.Noticef("wrong number of option descriptions for %s: expected %d, got %d", c.name, len(opts), len(c.optDescs))
```

Comment:
oops this and the previous one were committed by mistake (nee to patch go-flags before tweaking groups; leaving that for a later pass)